taker-frontend/src/components/CloseButton.tsx

```diff
@@ -0,0 +1,129 @@
+                            border="0"
+                            d="flex"
+                            alignItems="center"
+                            justifyContent="space-between"
+                            pb={4}
+                        >
+                            <Button
+                                size="sm"
+                                colorScheme="red"
+                                onClick={() => {
+                                    console.log(`Settling CFD ${cfd.order_id}`);
+                                    settle({});
+                                    onClose();
+                                }}
+                                isLoading={isSettling}
+                            >
+                                Close
+                            </Button>
+                        </PopoverFooter>
+                    </PopoverContent>
+                </>)}
+            </Popover>
+        </Box>;
+    } else {
+        closeButton = <Box w={"45%"}>
+            <Popover
+                placement="bottom"
+                closeOnBlur={true}
+            >
+                {({ onClose }) => (<>
+                    <PopoverTrigger>
+                        <Button colorScheme={"blue"} disabled={disableCloseButton}>Commit</Button>
+                    </PopoverTrigger>
+                    <PopoverContent color="white" bg="blue.800" borderColor="blue.800">
+                        <PopoverHeader pt={4} fontWeight="bold" border="0">
+                            Non-Collaboratively Close your position
+                        </PopoverHeader>
+                        <PopoverArrow />
+                        <PopoverCloseButton />
+                        <PopoverBody>
+                            <Text>
+                                This will force-close your position if your counterparty cannot be reached. The exchange
+                                rate at {cfd.expiry_timestamp}
+                                will determine your profit/losses. It is likely that the rate will change until then.
+                            </Text>
+                        </PopoverBody>
+                        <PopoverFooter
+                            border="0"
+                            d="flex"
+                            alignItems="center"
+                            justifyContent="space-between"
+                            pb={4}
+                        >
+                            <Button
+                                size="sm"
+                                colorScheme="red"
+                                onClick={() => {
+                                    console.log(`Commiting CFD ${cfd.order_id}`);
+                                    commit({});
+                                    onClose();
+                                }}
+                                isLoading={isCommiting}
+                            >
+                                Non-Collaborative Close
```

Comment:
as discussed, let's try to keep the size of the button relatively short - perhaps "Commit" would be enough?

Reply:
I'd just call it `close`. I don't think it is clear to users what `commit` means.

Reply:
i went with force-close